Documentation/Debugging.md

```diff
@@ -0,0 +1,42 @@
+# Implementation Validation with Assurance
+
+The AEP SDK offers an extension to inspect quickly, validate, debug data collection, and experiences for any mobile app using the AEP SDK. We built Assurance to do the heavy lifting of getting an SDK implementation right, so app developers can focus on creating engaging experiences.
```

Comment:
`inspect quickly` -> `quickly inspect` (makes it lead into your list a little better)